src/app/screens/connectors/ConnectGaloy/index.tsx

```diff
@@ -168,26 +168,31 @@ export default function ConnectGaloy(props: Props) {
       submitLabel={t("galoy.actions.login")}
       submitLoading={loading}
       onSubmit={loginWithJwt}
+      description={
+        <Trans
+          i18nKey={"galoy.token.info"}
+          t={t}
+          values={{ label }}
+          components={[
+            // eslint-disable-next-line react/jsx-key
+            <a
+              href="https://wallet.mainnet.galoy.io"
+              className="underline"
+            ></a>,
+            // eslint-disable-next-line react/jsx-key
+            <br />,
+            // eslint-disable-next-line react/jsx-key
+            <b></b>,
+          ]}
+        />
+      }
     >
       {
         <div className="mt-6">
-          <Trans
-            i18nKey={"galoy.token.info"}
-            t={t}
-            values={{ label }}
-            components={[
-              // eslint-disable-next-line react/jsx-key
-              <a
-                href="https://wallet.mainnet.galoy.io"
-                className="underline"
-              ></a>,
-              // eslint-disable-next-line react/jsx-key
-              <br />,
-              // eslint-disable-next-line react/jsx-key
-              <b></b>,
-            ]}
-          />
-          <label htmlFor="jwt" className="block font-medium text-gray-700">
+          <label
+            htmlFor="jwt"
+            className="block font-medium text-gray-800 dark:text-white"
```

Comment:
was it intentional to change the light mode styles too?

Reply:
Yes I changed the label's font color so that is similar to the font color in connector forms. For other connector screens the `Textfield` component is used and there the label has a font color style is `text-gray-800` so I have used the same here too.